node/pkg/db/db.go

```
@@ -128,6 +128,17 @@ func (d *Database) StoreSignedVAA(v *vaa.VAA) error {
 	return nil
 }
 
+func (d *Database) HasVAA(id VAAID) bool {
```

This should return an error if it is anything other than `ErrKeyNotFound`.

---

fixed

---

I don't see a fix.